ext/snowcrash/test/test-ResourceParser.cc

```diff
@@ -488,7 +488,7 @@ TEST_CASE("Parse model with unrecognised resource", "[resource][model]")
     REQUIRE(resource.node.actions[0].examples[0].responses[0].description == "");
 }
 
-TEST_CASE("Parse named resource with lazy referencing", "[resource][model][issue][84][now]")
+TEST_CASE("Parse named resource with lazy referencing", "[resource][model][issue][84]")
```

Comment:
Why was the `[now]` tag removed?

Reply:
That tag was supposed to be temporary for testing a single test and I forgot to remove it before